snuba/migrations/autogeneration/diff.py

```diff
@@ -61,34 +70,72 @@ def generate_migration_ops(
     return (forwardops, [op.get_reverse() for op in reversed(forwardops)])
 
 
-def _is_valid_add_column(oldstorage: str, newstorage: str) -> tuple[bool, str]:
+def _migration_ops_to_migration(
+    forwards_ops: Sequence[AddColumn],
+    backwards_ops: Sequence[DropColumn],
+) -> str:
+    """
+    Given a lists of forward and backwards ops, returns a python class
+    definition for the migration as a str. The migration must be non-blocking.
+    """
+
+    forwards_str = (
+        "[" + ", ".join([f"operations.{repr(op)}" for op in forwards_ops]) + "]"
+    )
+    backwards_str = (
+        "[" + ", ".join([f"operations.{repr(op)}" for op in backwards_ops]) + "]"
+    )
+
+    return f"""
```

Comment:
Should we extract the below code into a separate helper functions which take forwards_str and backwards_str and return the generated migration class.